pkg/k8s/api.go

```diff
@@ -9,7 +9,7 @@ import (
 	"net/url"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: undo

Reply:
Ha! This will be hard to catch because VS Code is putting it in every time the file is saved. I will have to be more diligent in looking at the diffs.